bitchat/Models/CommandsInfo.swift

```diff
@@ -0,0 +1,73 @@
+//
+// CommandsInfo.swift
+// bitchat
+//
+// This is free and unencumbered software released into the public domain.
+// For more information, see <https://unlicense.org>
+//
+
+import Foundation
+
+// MARK: - CommandInfo Enum
+
+enum CommandInfo: Identifiable {
+    case block, clear, hug, message, slap, unblock, who, favorite, unfavorite
+    
+    var id: String { name }
+
+    var name: String {
+        String(describing: self)
+    }
+    
+    var aliases: [String] {
+        switch self {
+        case .message:
+            return ["/m", "/msg"]
+        case .who:
+            return ["/w", "/who"]
+        default:
+            return ["/\(name)"]
+        }
+    }
+    
+    var primaryAlias: String { aliases.first ?? "" }
```

Comment:
the whole point of using an enum is to enumerate all the possible cases to avoid unnecessary optionality and string conversions as much as possible :)